### DISCUSSION TITLE
Unable to use fork library with latest version (1.3.2)

### DISCUSSION THREAD

**Discussion by yousrasd** [react-native-maps/react-native-maps#4543] [Q&A]
Hello,

Our project is using a fork of react-native-maps. Previously, our fork was using react-native-maps 0.30. I updated our fork by synchronizing it with the latest version of react-native-map master branch. So now, it is using react-native-maps v1.3.2.
Following this update, our app does not start. Im running into the following error:

`error: Error: While trying to resolve module `react-native-maps` from file `/path/projectname/folderA/folderB/file.js`, the package `/path/projectname/**node_modules/react-native-maps/package.json**` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/path/projectname/node_modules/react-native-maps/lib/index.js`. Indeed, none of these files exist`

When I check our node modules, I indeed do not see a **lib** folder
![Screenshot 2022-12-01 at 3 02 25 PM](https://user-images.githubusercontent.com/63681471/205174571-2d844519-fee7-4201-b011-fac89937b361.png)

I tried changing our project to use the main lib instead of the fork (react-native-maps/react-native-maps), and then I reinstalled the packages (yarn install). Then rechecked the node_modules, and Indeed I see a lib folder, and the app starts successfully.
![Screenshot 2022-12-01 at 3 00 51 PM](https://user-images.githubusercontent.com/63681471/205174736-8d5c7a1c-0d35-417a-a4da-5f3df15b2d5b.png)

Update: I tried forking this lib under my personal account (https://github.com/yousrasd/react-native-maps). When I use the master branch (which is up to date with this lib), same thing happens when I add this fork in the project: I run into  'this package itself specifies a `main` module field that could not be resolved' error.
I created a [branch](https://github.com/yousrasd/react-native-maps/tree/test) 'test' that points to an old version of this lib (v0.30.1). When I use this branch in our project, the app **STARTS**! So now I confirm that there is something in one of the latest versions (versions 1.0.0 and up) that is causing me this error. I see that now this lib is using Typescript, could it be related?

Is anyone running into the same issue?
If you have an idea on what could be causing this, please let me know. Any help would be appreciated.

**Comment by ravindraguptacapgemini** [comment]
@Simon-TechForm Can you please check this issue, getting the same issue on forked branch
